tasklauncher/spring-cloud-starter-stream-sink-tasklauncher-cloudfoundry/README.adoc

```diff
@@ -0,0 +1,45 @@
+//tag::ref-doc[]
+= TaskLauncher Cloud Foundry Sink
+
+The TaskLauncher-CloudFoundry sink consumes a `TaskLaunchRequest` message, deploys
+and launches a task using the Cloud Foundry Spring Cloud Deployer.
+
+Any message received by the sink, that does not contain a `TaskLaunchRequest`
+will log an error.
+
+== Using the TaskLauncher
+The task launcher will use the --uri from the TaskLaunchRequest to obtain the
+task artifact to deploy, as well as the properties and arguments to execute the
+task.
+An example of this using Spring Cloud Data Flow would look like this:
+
+*Create The Stream*
+
+`stream create w7 --definition "http --port=9000 | taskLaunchRequestTransform
```

Comment:
Should make it clear that the app we're talking about here is `cftask`. Also, should cross-reference the `taskLaunchRequestTransform` processor.